### DISCUSSION TITLE
Support given coords take the coordinate city boundary

### DISCUSSION THREAD

**Discussion by michaelr231** [osm-search/Nominatim#2533] [Geocoding around the world]
hey, 
if i have some issue, for example if i want to get the city boundary for this coordinate [ lat: 32.0852999, lon: 34.78176759999999] 
(tel aviv city) if i search reverse method with nominatim with zoom level 12\10 ( city, town ) it will not give me the city boundary that i am looking for, only if i use zoom level 8, but i want like use the zoom level dynamiclly.
there is any way that i send a coordinate and it will fetch me the city boundary that its included in ( polygon boundary coordinates )
?
thanks for any idea's

**Comment by mtmail** [comment]
Looking at the address hierarchy table with zoom level 18 https://nominatim.openstreetmap.org/ui/details.html?osmtype=W&osmid=239790649&class=highway it seems the city boundary is mapped weird, or maybe not as I would map it. 

https://www.openstreetmap.org/relation/2731722 is a relation without admin_level and not linked to the city node https://www.openstreetmap.org/node/278472603 via relation member or having the same wikidata tag.

Compared to Haifa https://www.openstreetmap.org/relation/1387888 and https://www.openstreetmap.org/node/1656107649

or Beer Sheva https://www.openstreetmap.org/relation/1377264

So specifically with Tel-Aviv changing the relation tags should help. https://openstreetmap.community/ lists multiple contact channels per region for local mappers, I don't want to second guess if the current mapping is intentional or a mistake.

In general Nominatim only assigns one rank to a place and that can cause issues if a city is also county/district/state of the same name or in some cases a city is also the same as the country. In those cases, e.g. Andorra the zoom parameter or structured forward search (city=Andorra) fail.

**Comment by mtmail** [comment]
I corrected https://www.openstreetmap.org/relation/1382494 where some relation members (outer boundary ways) were in the wrong order. It's not visible on the map, it renders fine, but Nominatim treated it as broken polygon. Now the node https://www.openstreetmap.org/node/278472603 is linked to a relation again. That might have fixed this specific query.

https://nominatim.openstreetmap.org/reverse?lat=32.0852999&lon=34.78176759999999&zoom=10&format=jsonv2

Still Tel-Aviv city vs district vs landuse seems to come down to non-trivial political definitions. Wikidata has two separate entries but only one Wikipedia article for example.

**Reply by michaelr231** [reply]
Hey where can i make corrections too?

**Reply by michaelr231** [reply]
@mtmail

**Reply by mtmail** [reply]
I used https://josm.openstreetmap.de/wiki/Help/Dialog/RelationEditor and reordered the members until the arrows all pointed in the same direction. I'm not sure that wasn't the root cause after all. Nominatim (https://nominatim.openstreetmap.org) needed to reindex the places in that area and one way to trigger that is changing a geometry. Reordering members is seen as change, better than moving a way or node around on the map.

**Comment by lonvia** [comment]
The landuse=residential polygon is imho wrong and should be deleted. It includes a huge park, industrial areas, a large busstation and a motorway, none of which is a residential area.
